src/cap-description.adoc

```diff
@@ -189,7 +192,7 @@ reserved values as if it were 0b00000 (no permissions). Future extensions may as
 meanings to the reserved bit patterns, in which case <<GCPERM>> is allowed to report a
 non-zero value.
 
-A {cap_rv64_perms_width}-bit vector encodes the permissions and <<m_bit>> when MXLEN=64. In
+A {cap_rv64_perms_width}-bit vector encodes the permissions MXLEN=64. In
```

Comment:
It looks like the table [here](https://github.com/riscv/riscv-cheri/pull/271/files#diff-2ac200119dffa8d1c8c860fa8f9d6614dc07cf4563a3cc1820cceb25152a12fbR212) has the M bit commented out, so we are no longer showing the M bit anywhere in any diagram or table. Am I missing something? It will be hard to understand where the M bit is encoded in the capability format for MXLEN=64

Reply:
I initially meant that I thought these figures did not have the M bit separately because I did not see a change for that here
![Screenshot from 2024-06-03 13-56-44](https://github.com/riscv/riscv-cheri/assets/3746901/ace427b0-39ed-4219-862e-427781df9336)
But the M bit is indeed there (although there is a misalignment in the bit number labels!), so we can resolve this!